src/components/MoneyRequestConfirmationListFooter.tsx

```diff
@@ -495,6 +504,25 @@ function MoneyRequestConfirmationListFooter({
             shouldShow: shouldShowTax,
             isSupplementary: true,
         },
+        {
+            item: (
+                <MenuItemWithTopDescription
+                    key="attendees"
+                    shouldShowRightIcon
+                    title={iouAttendees?.map((item) => item.displayName ?? item.login).join(', ')}
+                    description={`${translate('iou.attendees')} ${
+                        iouAttendees?.length && iouAttendees.length > 1 ? `\u00B7 ${formattedAmountPerAttendee} ${translate('common.perPerson')}` : ''
```

Comment:
Using `\u00B7` was intentional?

Reply:
Yes, it is unicode for middle dot symbol